<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ugk_planner</class>
 <widget class="QDialog" name="ugk_planner">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>998</width>
    <height>709</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>UGK_planneer</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QTableWidget" name="tableWidget">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>10</y>
     <width>601</width>
     <height>671</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextEdit" name="textEdit">
   <property name="geometry">
    <rect>
     <x>14</x>
     <y>11</y>
     <width>341</width>
     <height>71</height>
    </rect>
   </property>
   <property name="html">
    <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'.AppleSystemUIFont'; font-size:13pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p align=&quot;center&quot; style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:24pt;&quot;&gt;Введите ФИО&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>100</y>
     <width>351</width>
     <height>571</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QComboBox" name="comboBox_2">
      <property name="editable">
       <bool>false</bool>
      </property>
      <property name="currentText">
       <string>Выберите кафедру</string>
      </property>
      <property name="maxVisibleItems">
       <number>20</number>
      </property>
      <item>
       <property name="text">
        <string>Выберите кафедру</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Кафедра специальное фортепиано</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Кафедра бщего фортепиано</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Кафедра оркестровых струнных инструментов</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Кафедра оркестровых духовых и ударных инструментов</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Кафедра народных инструментов</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Кафедра камерного ансамбля</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Кафедра сольного пения</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Кафедра музыкального театра</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Кафедра хорового дирижирования</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Кафедра композиции</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Кафедра гуманитарных дисциплин</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Кафедра звукорежиссуры</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Кафедра истории музыки</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Кафедра теории музыки</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Кафедра теории музыки</string>
       </property>
      </item>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="comboBox">
      <property name="editable">
       <bool>false</bool>
      </property>
      <property name="maxVisibleItems">
       <number>2</number>
      </property>
      <item>
       <property name="text">
        <string>Выберите зал</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Большой зал</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Малый зал</string>
       </property>
      </item>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="comboBox_3">
      <property name="editable">
       <bool>false</bool>
      </property>
      <property name="maxVisibleItems">
       <number>2</number>
      </property>
      <item>
       <property name="text">
        <string>Выберите рояль</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Steinway&amp;Son`s 1</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Steinway&amp;Son`s 2</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Petrof 1</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Petrof 2</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>C.Bechstein 1</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>C.Bechstein 2</string>
       </property>
      </item>
     </widget>
    </item>
    <item>
     <widget class="QTextEdit" name="textEdit_2">
      <property name="html">
       <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'.AppleSystemUIFont'; font-size:13pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p align=&quot;center&quot; style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-size:24pt;&quot;&gt;Описание события&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCalendarWidget" name="calendarWidget"/>
    </item>
    <item>
     <widget class="QDateTimeEdit" name="dateTimeEdit">
      <property name="buttonSymbols">
       <enum>QAbstractSpinBox::UpDownArrows</enum>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="pushButton">
      <property name="whatsThis">
       <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <property name="text">
       <string>Сохранить</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
